tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -52,50 +101,197 @@ class SQLPlanParserSuite extends FunSuite with BeforeAndAfterEach with Logging {
           .filter($"b" < 100)
           .sort($"b")
       }
+      val pluginTypeChecker = new PluginTypeChecker()
+      val app = createAppFromEventlog(eventLog)
+      assert(app.sqlPlans.size == 1)
+      app.sqlPlans.foreach { case (sqlID, plan) =>
+        val planInfo = SQLPlanParser.parseSQLPlan(plan, sqlID, pluginTypeChecker, app)
+        assert(planInfo.execInfo.size == 11)
+        val wholeStages = planInfo.execInfo.filter(_.exec.contains("WholeStageCodegen"))
+        assert(wholeStages.size == 6)
+        // only 2 in the above example have projects and filters
```

Comment:
Nit: update comment to : `// only 2 in the above example have projects and filters and other 3 have sort.`

Reply:
updated